msal/src/Microsoft.Identity.Client/Features/ConfidentialClient/ConfidentialClientApplication.cs

```diff
@@ -114,7 +114,39 @@ public async Task<AuthenticationResult> AcquireTokenOnBehalfOfAsync(IEnumerable<
             Authority authorityInstance = Core.Instance.Authority.CreateAuthority(authority, ValidateAuthority);
             return
                 await
-                    AcquireTokenOnBehalfCommonAsync(authorityInstance, scopes, userAssertion, ApiEvent.ApiIds.AcquireTokenOnBehalfOfWithScopeUserAuthority)
+                    AcquireTokenOnBehalfCommonAsync(authorityInstance, scopes, userAssertion, ApiEvent.ApiIds.AcquireTokenOnBehalfOfWithScopeUserAuthority, false)
+                        .ConfigureAwait(false);
+        }
+
+        /// <summary>
+        /// Acquires token using On-Behalf-Of flow. (See https://aka.ms/msal-net-on-behalf-of)
```

Comment:
>(See https://aka.ms/msal-net-on-behalf-of) [](start = 52, length = 42)

pls. add the same link to the other obo flow methods. btw. add relevant links everywhere here. not just on the added methods.